autobuild/action.yml

```
@@ -3,6 +3,7 @@ description: 'Attempt to automatically build code'
 author: 'GitHub'
 inputs:
   token:
+    description: "Token for accessing /code-scanning/analysis endpoints (used to for status reporting)"
```

Same as in `analyze/action.yml`, and let's explicitly add the `required: false` block? 
```suggestion
    description: "GitHub token to use for authenticating with this instance of GitHub. The token needs the `security-events: write` permission."
    required: false
```